tests/devapps/MauiApps/MauiAppWithBroker/MainPage.xaml.cs

```diff
@@ -41,12 +45,36 @@ async private void OnSignInClicked(object sender, EventArgs e)
 
         private async void SignOutButton_Clicked(object sender, EventArgs e)
         {
-            await PCAWrapper.Instance.SignOut().ContinueWith(async (t) =>
+            await PCAWrapper.Instance.SignOutAsync().ContinueWith(async (t) =>
             {
                 await ShowMessage("Signed Out", "Sign out complete").ConfigureAwait(false);
             }).ConfigureAwait(false);
         }
 
+        // Call the web api. The code is left in the Ux file for easy to see.
+        private async Task<string> CallWebAPIWithToken(AuthenticationResult authResult)
+        {
+            try
+            {
+                //get data from API
+                HttpClient client = new HttpClient();
+                // create the request
+                HttpRequestMessage message = new HttpRequestMessage(HttpMethod.Get, "https://graph.microsoft.com/v1.0/me");

```

Comment:
graph endpoint can be made a string constant? if it is used elsewhere

Reply:
Not used elsewhere.